include/vdef.h

```diff
@@ -266,6 +266,8 @@ typedef struct {
 #define Tstr(s)		(/*lint -e(446)*/ (txt){(s), (s) + strlen(s)})
 #define Tstrcmp(t, s)	(strncmp((t).b, (s), Tlen(t)))
 
+#define TFOREACH(c, t) for ((c) = (t).b; (c) < (t).e; (c)++)
```

Comment:
Nice find.
Could we instead go for `#define Tforeach(c, t)` ? or `#define TXT_FOREACH(c, t)` ?
It was not obvious to me what the macro was meant to iterate over.

Reply:
T'was not obvious to me either so I went with T like in other txt-related macros and FOREACH like in other iterator macros.

`TXT_FOREACH()` was my alternative.

The pun was of course intended.

Reply:
My useless contribution on bikeshed colors: `TXT_FOREACH_CHAR`
Please not `TFOREACH`, for consistency I'd otherwise propose `Tforeach` or `Titer`

Reply:
Never too late to rename for the better ;)